lib/internal/console/constructor.js

```diff
@@ -54,6 +55,8 @@ const kBindStreamsLazy = Symbol('kBindStreamsLazy');
 const kUseStdout = Symbol('kUseStdout');
 const kUseStderr = Symbol('kUseStderr');
 
+const optionsMap = new WeakMap();
```

Comment:
Yes, symbol properties are observable and IMO this should not be observable / should be handled as a private property.

Reply:
Why the extra magic?

Reply:
Sorry, I am a bit lost. In what way extra magic? Do you have an alternative solution?

Reply:
I mean, why do we go out of our way to hide this? We don’t it for other properties, and I can’t find a reason here?

Reply:
> We mainly did not do this for performance reasons but we have multiple places where we use this pattern as well.

Yes, and I think those are pretty good examples of the decreased readability that comes with it. Not quite as bad in this case here, though.

> I guess we both just have different opinions about private members in general?

Yes, definitely – to me, they seem like the biggest mistake the language could be making, and I would want us to avoid adopting them as much as possible. I guess there’s not much point in arguing this here, though…